salt/redmine/init.sls

```diff
@@ -0,0 +1,98 @@
+{% from 'lib.sls' import create_user %}
+
+{% set user = 'redmine' %}
+{% set branch = '5.0-stable' %}
+{% set revision = 21783 %}
+
+include:
+  - apache
+  - apache.modules.passenger
+  - mysql
+  - rvm
+
+{{ create_user(user) }}
+
+/home/{{ user }}/public_html:
+  file.directory:
+    - user: {{ user }}
+    - group: {{ user }}
+    - require:
+      - user: {{ user }}_user_exists
+
+# https://redmine.org/projects/redmine/wiki/HowTo_Install_Redmine_50x_on_Ubuntu_2004_with_Apache2
+redmine dependencies:
+  pkg.installed:
+    - pkgs:
+      - ruby-dev
+      - libmysqlclient-dev
+      # https://www.redmine.org/projects/redmine/wiki/redmineinstall#Optional-components
+      - ghostscript
+      - imagemagick
+
+deploy redmine:
```

Comment:
When I deploy (with `test=True`) I don't get any duplicate state error. I changed it to plain `redmine`, so that the `require: pkg: redmine` wouldn't have any spaces, which seems error-prone.

Reply:
Plain `redmine` clashes with the MySQL database creation sls. This only runs for me because the DB is configured in `pillar/private` which might be the discrepancy?

Clashing line https://github.com/open-contracting/deploy/blob/main/salt/mysql/init.sls#L67

Reply:
Aha, I've renamed the database creation state to avoid unexpected clashes. I'll do the same for PosgreSQL on the main branch.